docs/events.md

```diff
@@ -52,6 +53,7 @@ There are several event types that you can record within the customer journey:
 | [`pageLoaded`](#pageloaded)                             | Web           | Yes           | Records a page load.
 | [`productViewed`](#productviewed)                       | E-commerce    | No            | Records a product view.
 | [`cartViewed`](#cartviewed)                             | E-commerce    | No            | Records a cart view.
+| [`cartModified`](#cartmodified)                         | E-commerce    | No            | Records a cart modify.
```

Comment:
`Records a shopping cart change.` Please, also update `Records a cart view.` to `Records a shopping cart view` to keep consistency.